pandas/tests/frame/methods/test_value_counts.py

```diff
@@ -144,3 +144,31 @@ def test_data_frame_value_counts_dropna_false(nulls_fixture):
     )
 
     tm.assert_series_equal(result, expected)
+
+
+def test_data_frame_value_counts_subset(nulls_fixture):
+    # GH 50829
+    df = pd.DataFrame(
+        {
+            "first_name": ["John", "Anne", "John", "Beth"],
+            "middle_name": ["Smith", nulls_fixture, nulls_fixture, "Louise"],
+        },
+    )
+    result = df.value_counts("first_name")
+    expected = pd.Series(
+        data=[2, 1, 1],
+        index=pd.Index(["John", "Anne", "Beth"], name="first_name"),
+    )
+
+    tm.assert_series_equal(result, expected)
+
+    df = pd.DataFrame(
+        {100: [2, 100, 5, 9], 200: [2, 6, 2, 6], 300: [4, 6, 2, 1]},
+    )
+    result = df.value_counts([200])
```

Comment:
I think lists already have test coverage, my earlier request was to test `df.value_counts(200)`. This will require the fix for the implementation I mentioned in https://github.com/pandas-dev/pandas/pull/50955/#pullrequestreview-1274174555.

Also, instead of duplicating the code for the tests, can you parameterize this; e.g. add

    @pytest.mark.parametrize("columns", (["first_name", "middle_name"], [0, 1]))

to the top of the test and then use `columns[0]` and `columns[1]` in the original test where appropriate.

Reply:
My bad. I'll be more than happy to work on this.